gammapy/spectrum/tests/test_spectrum_analysis.py

```diff
@@ -19,6 +21,12 @@
     SpectrumFit,
 )
 
+try:
```

Comment:
This try ... except code that checks for WStat is present twice.
Can you please implement a `SHERPA_GT_4_8` in `gammapy.utils.testing`.
(I haven't looked how `from astropy.utils.compat import NUMPY_LT_1_9` is implemented ... can you check and do the same?)